data/showcase/taxipp.yml

```diff
@@ -0,0 +1,14 @@
+title: TaxIPP
+description:
+  fr: Modèle de micro-simulation statique du système socio-fiscal français connectant OpenFisca à des bases de données administratives sur la population française.
+  en: ??
```

Comment:
Pour cette ligne : "Microsimulation model of the French tax and benefit system connecting Openfisca to administrative data"